### PR TITLE
Unify to single CI platform

### PR DESCRIPTION
**Pull request by atifaziz**
This PR addresses #602 by unifying on AppVeyor as the single CI platform.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,24 +1,64 @@
...
+  matrix:
+    only:
+      - image: Visual Studio 2019
+  environment:
+    IMAGE_NAME: win
+  deploy:
+  - provider: NuGet
+    server: https://www.myget.org/F/morelinq/api/v2/package
+    api_key:
+      secure: fhGwXyO35FSshRzs5GWmF1LJTrd1sIqmS/jNCSfO2LfOciuYAKiXuFMYZFGiTAl+
```
Comment (fsateler) [inline appveyor.yml:20]: Isn't this supposed to be secret?

**Review by fsateler — COMMENTED** [review]
Hi!

This looks interesting. Did you unify on appveyor only because it was easier? Or for another reason?

I ask, because Github Actions seems to be the new kid on the block everyone is talking about :wink:

File: appveyor.yml (hunk 1 above)
Comment (atifaziz) [reply appveyor.yml:20]: Yes, but the secret is [secured](https://ci.appveyor.com/tools/encrypt) (as suggested in [Publishing to NuGet feed](https://www.appveyor.com/docs/deployment/nuget/)) and something only AppVeyor's build runner can decrypt.

**Comment by atifaziz** [comment]
> Did you unify on appveyor only because it was easier?

Yes, as I have already some experience with it.

> Or for another reason?

Time. 😄 

> I ask, because Github Actions seems to be the new kid on the block everyone is talking about 😉

I know and I have tried it with just one project so I am barely proficient in it. Everything system needs time getting used to with some trial and error. I have been using AppVeyor for ages and it has served me well. It now supports Linux and macOS images so it was the path of least friction. I am happy to move to GitHub Actions if someone well-versed in it can help with the setup and maintenance. However, given that issue #602 has been open since over a year and no one really picked it up, I just wanted to get this sorted out for now so we can move to more interesting (less infrastructural) work.

**Review by fsateler — APPROVED** [review]
LGTM

**Pull request merged**